.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: c
+sudo: required
+dist: xenial
+install:
+    - sudo apt-get install -y python3 python3-pip
+    - sudo pip3 install scapy-python3
+    - git submodule init && git submodule update
+before_script:
+    - sudo sysctl -w net.ipv4.tcp_fastopen=5
+script:
+    - mkdir build && cd build && cmake .. && make && sudo make test
```

Comment:
why do you need `sudo` for the tests? (I didn't check what was done there but I guess you don't need `sudo`)

Reply:
The tests need to capture packets on loopback iface (I could add the capabilities but easier enough like this)

Reply:
:+1: